packages/core/src/reporters/progress-reporter.ts

```diff
@@ -1,10 +1,10 @@
 import { MatchedMutant, MutantResult } from '@stryker-mutator/api/report';
 
-import ProgressBar from './progress-bar';
-import ProgressKeeper from './progress-keeper';
+import { ProgressBar } from './progress-bar';
+import { ProgressKeeper } from './progress-keeper';
 
-export default class ProgressBarReporter extends ProgressKeeper {
-  private progressBar: ProgressBar;
+export class ProgressBarReporter extends ProgressKeeper {
+  private progressBar?: ProgressBar;
```

Comment:
so progressBar is optional? 🤔 potentially dangerous, unless you know what you are doing

Reply:
True, it is dangerous, but your reasoning the wrong way around. It was always optional because we didn't set the value in the constructor or inline here. However, we never got warnings when we used it without a null-check because typescript didn't warn us. 

By enabling [`--strictPropertyInitialization`](https://www.typescriptlang.org/tsconfig#strictPropertyInitialization) I now got a compile error that I fixed by adding the `?`. This triggered further type errors in the places we used it without a null-check, that's how I fixed a small bug :).